packaging/windows/sharedframework/bundle.wxs

```diff
@@ -42,6 +42,7 @@
                        ProductName="$(var.Crt_ProductName)" 
                        Size="$(var.Crt_Size)" 
                        Version="$(var.Crt_Version)" />
+        <ExitCode Behavior="success" Value="1638" />
```

Comment:
Evidence indicates that corrupted installation of a version that not in that link is what results in the failure.